crates/model/src/order.rs

```diff
@@ -359,33 +360,91 @@ impl OrderData {
 
 /// An order as provided to the POST order endpoint.
 #[serde_as]
-#[derive(Clone, Debug, Default, Deserialize, Eq, PartialEq, Serialize)]
+#[derive(Clone, Debug, Default, Deserialize, Eq, PartialEq, Serialize, ToSchema)]
 #[serde(rename_all = "camelCase")]
+#[schema(example = json!({
```

Comment:
Does it get checked that this type actually satisfies the interface during compilation?

Reply:
I think it would be better to specify example value closer to the field or even type. Currently it would be a bit difficult to maintain any changes.

Reply:
Is this even necessary, given that you specify example values in each field's definition?